newbank/client/ExampleClient.java

```diff
@@ -1,8 +1,5 @@
 package newbank.client;
 
-import newbank.server.NewBank;
```

Comment:
These were only imported when I tried some stuff out for the "10. Exit" request. These were not there previously and are not needed